libqtile/backend/x11/core.py

```diff
@@ -185,10 +185,7 @@ def finalize(self) -> None:
         self.conn.finalize()
 
     def get_screen_info(self) -> list[ScreenRect]:
-        ps = self.conn.pseudoscreens
-        if self.qtile:
-            self._xpoll()
```

Comment:
Is the removal of this call to _xpoll perhaps a problem?

Reply:
No, that's one of the goals of the patch series though.

We call get_screen_info() *after* we get the ScreenChangeNotify event, as a result of it. I suspect that the code here is wrong: https://github.com/qtile/qtile/pull/5218/files#diff-ec6963b4884cc405393d63da145f2ed420915dd373a5b6689a1fc726ed237487R423 or not getting called or something. But it's copy pasted from the original code, so it should be the same...